.github/workflows/build.yml

```diff
@@ -39,6 +41,8 @@ jobs:
       runs-on: ubuntu-latest
       steps:
         - uses: actions/checkout@main
```

Comment:
I think this `feature-check` can be removed since we have a better version of it now (that checks each crate instead of the workspace).